leetcodePython/twopointers_42.py

```diff
@@ -0,0 +1,77 @@
+from typing import List
+
+class SolutionBruteForce:
+    def trap(self, height: List[int]) -> int:
+        rain = 0
+        size = len(height)
+
+        '''
+                        *
+        *       #       *
+        *       *       *
+        l       i       r
+        '''
```

Comment:
For every single index `i`, the trap water is based on:
1. max height of left wall until `i`.
2. max height  of right wall until `i`.
3. lower wall of the left wall & right wall